components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -149,6 +148,8 @@ enum ActionType {
   // Change length
   SHORTEN,
   EXPAND,
+  CONVERSATION_STARTER,
+  SUGGESTION,
```

Comment:
could you clarify what the difference is between a starter and a suggestion? are starters only be used when making the first prompt in the chat, and suggestions used for follow-ups?